runtime/src/commonMain/kotlin/pbandk/json/MessageExtensions.kt

```diff
@@ -0,0 +1,9 @@
+package pbandk.json
+
+import pbandk.Message
+
+fun <T: Message> T.jsonMarshal(jsonConfig: JsonConfig = JsonConfig.DEFAULT) =
+    JsonWireMarshaller(jsonConfig).also{ it.writeMessage(this) }.toJsonString()
+
+fun <T : Message> Message.Companion<T>.jsonUnmarshal(data: String, jsonConfig: JsonConfig = JsonConfig.DEFAULT): T =
+    unmarshal(JsonWireUnmarshaller.fromString(data, jsonConfig))
```

Comment:
Nice to put these extensions only once, as opposed to each generated file

Reply:
I also like that all of the JSON-related functionality is segregated into separate files. I'm considering in the future splitting these out into a separate `pbandk-json` library for folks that don't use JSON and don't want the extra code and dependencies needed for JSON support.